src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        Color color = new ColorSupplier().getRandomColor();
```

Comment:
Creating a new instance of ColorSupplier every time may be inefficient if the getRandomColor method does not depend on instance state. Consider making ColorSupplier a class-level field or its method static if there's no state.

Reply:
fix comment